.tool-versions

```diff
@@ -1 +0,0 @@
-nodejs 18.18.2
```

Comment:
Don't think we need this file - seems to be used by "asdf", which we aren't using. I also didn't update the version number when we moved to node 18.20 - oops - but maybe it's better to remove it.